spec/spec_helper.rb

```diff
@@ -22,3 +22,7 @@
   config.order = "random"
   config.raise_errors_for_deprecations!
 end
+
+def rails_version
+  Rails::VERSION::STRING.split(".").first.to_i
```

Comment:
I think `Rails::VERSION::STRING` is the same as `Rails.version`, but I think what you need here is rather `Rails::VERSION::MAJOR`